docs/guide/ingress/annotations.md

```diff
@@ -14,51 +14,52 @@ You can add annotations to kubernetes Ingress and Service objects to customize t
+| [alb.ingress.kubernetes.io/healthcheck-timeout-seconds](#healthcheck-timeout-seconds)                 | integer                     |'5'| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/healthy-threshold-count](#healthy-threshold-count)                         | integer                     |'2'| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/unhealthy-threshold-count](#unhealthy-threshold-count)                     | integer                     |'2'| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/success-codes](#success-codes)                                             | string                      |'200' \| '12' | Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-type](#auth-type)                                                     | none\|oidc\|cognito         |none| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-idp-cognito](#auth-idp-cognito)                                       | json                        |N/A| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-idp-oidc](#auth-idp-oidc)                                             | json                        |N/A| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-on-unauthenticated-request](#auth-on-unauthenticated-request)         | authenticate\|allow\|deny   |authenticate| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-scope](#auth-scope)                                                   | string                      |openid| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-session-cookie](#auth-session-cookie)                                 | string                      |AWSELBAuthSessionCookie| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/auth-session-timeout](#auth-session-timeout)                               | integer                     |'604800'| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/actions.${action-name}](#actions)                                          | json                        |N/A| Ingress          | N/A           |
+| [alb.ingress.kubernetes.io/conditions.${conditions-name}](#conditions)                                | json                        |N/A| Ingress          | N/A           |
+| [alb.ingress.kubernetes.io/target-node-labels](#target-node-labels)                                   | stringMap                   |N/A| Ingress,Service  | N/A           |
+| [alb.ingress.kubernetes.io/mutual-authentication](#mutual-authentication)                             | json                        |N/A| Ingress          | Exclusive     |
+| [alb.ingress.kubernetes.io/multi-cluster-target-group](#multi-cluster-target-group)                   | boolean                     |N/A| Ingress, Service | Merge         |
```

Comment:
What do you recommend for this setting?

Reply:
I would see it as Exclusive behavior which means such annotation should only be specified on a single Ingress within IngressGroup or specified with same value across all Ingresses within IngressGroup. Otherwise, setting it on one of the ingress will affect all the ingresses within that group serving different services which may not be desirable.

Reply:
I need some help understanding Ingress behavior I think:

```
func (t *defaultModelBuildTask) buildTargetGroupBindingMultiClusterFlag(ing ClassifiedIngress, svc *corev1.Service) (bool, error) {
	enabled, err := t.getMultiClusterTgFlag(ing.Ing.Annotations)
	if err != nil {
		return false, err
	}

	if enabled {
		return true, nil
	}

	return t.getMultiClusterTgFlag(svc.Annotations)
}
```

This is the current source code. I think a classifed ingress is the whole ingress group. I think it makes sense to have the behavior be merge because this is just additional behavior, that doesn't break backwards compatibility so having the multicluster support expand to other ingresses isn't harmful.